src/components/attendee/Card/Card.module.scss

```diff
@@ -2,6 +2,8 @@
 @use "scss/attendee/space";
 @use "scss/attendee/theme";
 
+$button-height: 42px;
```

Comment:
Private constant, just for this file

```suggestion
$-button-height: 42px;
```

Reply:
done: https://github.com/sparkletown/sparkle/pull/3003/commits/07ec924f308486ffff85bb746fdba69810c19e56